gorush/metrics.go

```diff
@@ -81,42 +81,42 @@ func (c Metrics) Describe(ch chan<- *prometheus.Desc) {
 func (c Metrics) Collect(ch chan<- prometheus.Metric) {
 	ch <- prometheus.MustNewConstMetric(
 		c.TotalPushCount,
-		prometheus.GaugeValue,
+		prometheus.CounterValue,
 		float64(StatStorage.GetTotalCount()),
 	)
 	ch <- prometheus.MustNewConstMetric(
 		c.IosSuccess,
-		prometheus.GaugeValue,
+		prometheus.CounterValue,
 		float64(StatStorage.GetIosSuccess()),
 	)
 	ch <- prometheus.MustNewConstMetric(
 		c.IosError,
-		prometheus.GaugeValue,
+		prometheus.CounterValue,
 		float64(StatStorage.GetIosError()),
 	)
 	ch <- prometheus.MustNewConstMetric(
 		c.AndroidSuccess,
-		prometheus.GaugeValue,
+		prometheus.CounterValue,
 		float64(StatStorage.GetAndroidSuccess()),
 	)
 	ch <- prometheus.MustNewConstMetric(
 		c.AndroidError,
-		prometheus.GaugeValue,
+		prometheus.CounterValue,
 		float64(StatStorage.GetAndroidError()),
 	)
 	ch <- prometheus.MustNewConstMetric(
 		c.HuaweiSuccess,
-		prometheus.GaugeValue,
+		prometheus.CounterValue,
 		float64(StatStorage.GetHuaweiSuccess()),
 	)
 	ch <- prometheus.MustNewConstMetric(
 		c.HuaweiError,
-		prometheus.GaugeValue,
+		prometheus.CounterValue,
 		float64(StatStorage.GetHuaweiError()),
 	)
 	ch <- prometheus.MustNewConstMetric(
 		c.QueueUsage,
-		prometheus.GaugeValue,
```

Comment:
I think `queue usage` should not change to counter value type.

Reply:
Good point, that one should remain a gauge

Reply:
@chaosaffe Please revert this line. Thanks